torch/functional.py

```diff
@@ -799,6 +802,76 @@ def _unique_impl(input: Tensor, sorted: bool = True,
         tensor([[0, 2],
                 [1, 2]])
 
+        >>> a = torch.tensor([
+        ...     [
+        ...         [1, 1, 0, 0],
+        ...         [1, 1, 0, 0],
+        ...         [0, 0, 1, 1],
+        ...     ],
+        ...     [
+        ...         [0, 0, 1, 1],
+        ...         [0, 0, 1, 1],
+        ...         [1, 1, 1, 1],
+        ...     ],
+        ...     [
+        ...         [1, 1, 0, 0],
+        ...         [1, 1, 0, 0],
+        ...         [0, 0, 1, 1],
+        ...     ],
+        ... ])
+
+        If we call `torch.unique(a, dim=0)`, each of the tensors `a[idx, :, :]`
```

Comment:
This doesn't render well: https://docs-preview.pytorch.org/pytorch/pytorch/108292/generated/torch.unique.html#torch-unique
I think using `>>> # ...` would look better here?